src/clj/puppetlabs/services/jruby/jruby_puppet_schemas.clj

```diff
@@ -14,7 +13,7 @@
   "Schema defining the supported values for the JRuby CompileMode setting."
   (apply schema/enum supported-jruby-compile-modes))
 
-(def JRubyPuppetConfig
+(def CombinedJRubyPuppetConfig
```

Comment:
I think that would cause it have both `:max-borrows-per-instance` and `:max-requests-per-instance`